website/src/main/java/app/Main.java

```diff
@@ -7,9 +7,8 @@
 public class Main {
 
     public static void main(String[] args) {
-
         Javalin app = Javalin.create(config ->
-            config.addStaticFiles("src/main/resources/public", Location.EXTERNAL)
+            config.addStaticFiles("website/src/main/resources/public", Location.EXTERNAL)
```

Comment:
Are you running this from the project root?

Reply:
My IDE probably was.  It didn't seem to like that this was now a subproject.

Reply:
Okay, it doesn't matter much, whatever works for you 👍